tests/www/views/test_views_tasks.py

```diff
@@ -492,7 +496,7 @@ def test_code_from_db_all_example_dags(admin_client):
                 dag_id="example_bash_operator",
                 ignore_all_deps="false",
                 ignore_ti_state="true",
-                execution_date=DEFAULT_DATE,
+                dag_run_id=DEFAULT_DAGRUN,
```

Comment:
How did this ever work before? The view does not accept `execution_date` (not anymore, I think) but only `dag_run_id`.

Reply:
I guess we use mock client rather than actually call views here: https://flask.palletsprojects.com/en/2.2.x/testing/#sending-requests-with-the-test-client

https://github.com/apache/airflow/blob/f17e2ba48b59525655a92e04684db664a672918f/tests/www/views/conftest.py#L64-L65

Reply:
I think the mock client still executes the views (it simply skips the HTTP and directly calls the routers).